src/Nethermind/Nethermind.Evm.Test/TransactionSubstateTests.cs

```diff
@@ -136,7 +136,7 @@ private static IEnumerable<(byte[], string)> PanicFunctionTestCases()
                 "Reverted unknown panic code (0xff)");
 
             // Invalid case
-            yield return (new byte[] { 0xf0, 0x28, 0x8c, 0x28 }, "Reverted 0xf0288c28");
```

Comment:
Why changing this? Needs to test invalid Utf8 and make sure it gets output as hex

Reply:
> Why changing this? Needs to test invalid Utf8 and make sure it gets output as hex

Comes from https://github.com/NethermindEth/nethermind-security/commit/46de1e5cbc872af0a2b85781e506d16963ce3dc3 that was merged into this branch

Reply:
Is already in master, then this test was fixed, so please revert this line. Already test valid cases